src/main/java/core/basesyntax/AreaCalculable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculable {
```

Comment:
According to the checklist, interface names should not be verbs. Consider renaming the interface to something like 'AreaCalculator' to better represent its purpose as an entity that can calculate an area.